pubsub_workers_integrated/worker/site_worker_integrated.py

```diff
@@ -77,17 +78,20 @@ def scrape_data(self, miner, url):
             # into a datetime object (at midnight)
             "acquisition_date": datetime.combine(date(*[int(x) for x in miner.results['date_aquisition'].split("-")]), time),
         }
-        # if miner.results['date_publication'] is not None:
-        #    data["publication_date"] = datetime.combine(date(*[int(x) for x in miner.results['date_publication'].split("-")]), time)
 
         # Add the meta info
         meta_info = {
-            "references:": miner.results['references'],
+            "references": miner.results['references'],
             "search_keyword": miner.results['search_keyword'],
             "license": miner.results['license'],
             "extra_link": miner.results['extra_link']
         }
 
+        # Scielo specific meta-info
+        if 'title_translated' and 'abstract_translated' in miner.results:
+            meta_info['title_translated'] = miner.results['title_translated']
+            meta_info['abstract_translated'] = miner.results['abstract_translated']
```

Comment:
Should the meta info be added separately for translated titles and abstracts?

Reply:
fixed